### ISSUE TITLE
Develop a process and documentation for merging Greenkeeper PR's

### ISSUE THREAD

**Issue by rileytaylor** [code-corps/code-corps-ember#458]
This issue is for tracking and discussion of a process for working through the Greenkeeper PR's.

The eventual output of this will be a documentation file that discusses the process so that contributors can follow it and keep things updated and our PR count low (as of this writing, 75% of our 20 PR's are greenkeeper).
## Process (will edit this as discussion occurs)
1. Pull down the branch
2. Check the project's changelog for any breaking changes
3. Any steps that the project requires on install/update
4. Build, run and verify tests
5. Verify things look good: check the pages and components that make use of the library to ensure they are still working as expected.
6. Rebase, squash and merge
## Points of interest
1. Could be nice to automate this further with a greenkeeper branch of some sort that deploys somewhere for us to test with. That would give us a dedicated branch to manage code changes necessary for any major package updates and take care of breaking changes.
## Background

The tl;dr of [Greenkeeper](https://greenkeeper.io/) is that it creates pull requests with the updated versions of packages from npm that exist in our project.json. It automates through a commit that builds the project with the updated packages and then it will create a PR if that CI build passes. 

Theoretically, we could just merge from there, but it's always best to check with the packages log, run it on a machine, and double check everything works in the browser.

**Comment by JoshSmith** [comment]
I would consider adding a step to check the project's changelog to see if anything really significant (read: breaking change) has happened in this update.

**Comment by JoshSmith** [comment]
This is also possible to tell partly via version number if the project is using semver well.

**Comment by rileytaylor** [comment]
@JoshSmith That's a good step, I'll add that

**Comment by WenInCode** [comment]
This seems to cover mostly everything I do when reviewing the greenkeeper PRs. I normally also check to see where the corresponding addon is being used and verify that everything is the depends on it is still working as intended. This is easier said then done for a few things though.

**Comment by rileytaylor** [comment]
@WenInCode That's what I was going for with one of the steps, but i updated it to be more explicit.

**Comment by JoshSmith** [comment]
`Re-run and initializing` => `any`

**Comment by JoshSmith** [comment]
> Rebase on Develop and merge

=>

> Rebase, squash and merge

**Comment by JoshSmith** [comment]
@rileytaylor what's the status of this?

**Comment by rileytaylor** [comment]
@JoshSmith I haven't done anything on this yet, I can work on it this week.

**Comment by JoshSmith** [comment]
@rileytaylor isn't this mostly just taking what you have written above and PRing with some markdown changes?

Also, this seems low priority relative to others. I don't know that I would advocate for working on it so quickly unless it's a quick task.
